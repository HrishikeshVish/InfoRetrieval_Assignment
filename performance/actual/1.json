["https://archive.org/details/MSNBCW_20190923_100000_Morning_Joe#start/5379/end/5414", "https://archive.org/details/FOXNEWS_20100205_100000_The_OReilly_Factor#start/275/end/310", "https://archive.org/details/FOXNEWS_20100205_040000_The_OReilly_Factor#start/275/end/310", "https://archive.org/details/MSNBCW_20170524_070000_All_In_With_Chris_Hayes#start/3351/end/3386", "https://archive.org/details/CNN_20100916_170000_CNN_Newsroom#start/1839/end/1874", "https://archive.org/details/FOXNEWSW_20130322_200000_Your_World_With_Neil_Cavuto#start/353/end/388", "https://archive.org/details/MSNBCW_20191206_080000_All_In_With_Chris_Hayes#start/1941/end/1976", "https://archive.org/details/MSNBCW_20191224_190000_MSNBC_Live_With_Katy_Tur#start/779/end/814", "https://archive.org/details/CNNW_20180730_060000_CNN_Newsroom_Live#start/2038/end/2073", "https://archive.org/details/MSNBCW_20190823_000000_All_In_With_Chris_Hayes#start/688/end/723"]