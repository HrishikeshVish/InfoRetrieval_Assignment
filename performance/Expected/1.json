["https://archive.org/details/CNNW_20150925_090000_Early_Start_With_John_Berman_and_Christine_Romans#start/290/end/325", "https://archive.org/details/FOXNEWSW_20130322_200000_Your_World_With_Neil_Cavuto#start/353/end/388", "https://archive.org/details/CNNW_20150925_090000_Early_Start_With_John_Berman_and_Christine_Romans#start/2071/end/2106", "https://archive.org/details/CNNW_20150924_040000_CNN_Newsroom_Live#start/5368/end/5403", "https://archive.org/details/CNN_20100916_170000_CNN_Newsroom#start/1839/end/1874", "https://archive.org/details/FOXNEWSW_20150529_100000_FOX_and_Friends#start/8232/end/8267", "https://archive.org/details/MSNBCW_20170524_070000_All_In_With_Chris_Hayes#start/3351/end/3386", "https://archive.org/details/FOXNEWS_20100205_100000_The_OReilly_Factor#start/275/end/310", "https://archive.org/details/FOXNEWS_20100205_040000_The_OReilly_Factor#start/275/end/310", "https://archive.org/details/CNNW_20180730_060000_CNN_Newsroom_Live#start/2038/end/2073"]